docs/rate-limit-and-backpressure.md

```diff
@@ -0,0 +1,122 @@
+# Rate Limit and Backpressure
+
+Here we compare 2 concepts in RisingWave used to keep barrier latency low.
+
+## Why keep Barrier Latency Low?
+
+1. Commands such as `Drop, Cancel, Alter` depend on barriers.
+   When barrier latency is high, it also means these commands take a long time to run.* [1]
+2. High Barrier Latency means Low Data Freshness. User gets stale results.
+
+[1] This could potentially be resolved by decoupling `Drop, Cancel`: https://github.com/risingwavelabs/risingwave/issues/13396
+
+## Why can Barrier Latency be High?
+
+The data between barriers takes a long time to process.
+This can be due to high amplification, UDF latency, or other reasons.
+
+By default, Barriers have an interval of 256ms.
```

Comment:
It should be 1s. https://github.com/risingwavelabs/risingwave/blob/3a08273fd65aaa81db216e67759bb13c19a10aa8/src/config/example.toml#L208